_attributes/common-attributes.adoc

```diff
@@ -56,6 +56,7 @@ endif::[]
 :secondary-scheduler-operator-full: Secondary Scheduler Operator for Red Hat OpenShift
 :secondary-scheduler-operator: Secondary Scheduler Operator
 :descheduler-operator: Kube Descheduler Operator
+:cli-manager: OpenShift CLI Manager
```

Comment:
Official name is CLI Manager without OpenShift

Reply:
Should it be CLI Manager Operator?